### ISSUE TITLE
Auth flow for redux navigation

### ISSUE THREAD

**Issue by rawris** [spencercarli/react-navigation-auth-flow#2]
Hi Spencer,

A great tutorial indeed but can you make the same auth flow for react-navigation and redux together.

Thanks

**Comment by daniel-swain** [comment]
Yeah, I second this. I'm trying to integrate this tutorial into a current app, but I'm having a hard time accessing the router/getStateForAction properties of the main navigator

**Comment by eballeste** [comment]
@rawris @daniel-swain did you guys figure out how to setup the reducer for the root Navigator?

**Comment by albinhubsch** [comment]
@eballeste @rawris @daniel-swain 
I solved this by by adding a third screen to SignedOut called Splash that I connected to the redux store. From there I then navigate to either SignedIn or login/signup. Maybe not a perfect solution but it works.

**Comment by eballeste** [comment]
@albinhubsch thanks! I ended up doing kind of doing the same thing but instead I use a loader screen with an animation that then either takes you to the SignedIn or SignedOut screen.
